internal/hud/view/view.go

```diff
@@ -31,16 +31,22 @@ type Resource struct {
 	CurrentBuildStartTime time.Time
 	CurrentBuildLog       string
 
+	// Relevant to k8s resources (maybe should accomplish via interface?)
 	PodName         string
 	PodCreationTime time.Time
 	PodStatus       string
 	PodRestarts     int
 	Endpoints       []string
-	PodLog          string
+
+	// Relevant to docker-compose resources (maybe should accomplish via interface?)
+	IsDCManifest bool
```

Comment:
Again I would advocate for making this a separate type, but we previously discussed this and you're foregoing that for now.